src/renderer/components/Native/EngineClient.vue

```
@@ -416,10 +421,12 @@ export default {
 			// tasks (BT/ED2K/ADC/Gnutella/G2/giFT) take minutes to warm up and a
 			// force-pause aborts that warm-up; the cooldown then refires before
 			// the swarm can rebuild, leaving speed permanently at zero. Filter
-			// down to kinds that actually benefit before consulting the evaluator
-			const eligibleTasks = tasks.filter((task) =>
-				taskBenefitsFromLowSpeedRecovery(task),
-			);
+			// skip tasks whose totalLength is still zero
+			const eligibleTasks = tasks.filter((task) => {
+				if (!taskBenefitsFromLowSpeedRecovery(task)) return false;
```

Biome (2.4.16)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)